pages/movies/[movieId]/index.tsx

```diff
@@ -31,13 +32,16 @@ export default function MovieDetails() {
                             <Col>
                                 <Image className={"w-75 p-3"} style={{borderRadius: 25}} src={"https://image.tmdb.org/t/p/w500" + movieDetails.poster_path} alt="image of movie" />
                             </Col>
-                            <Col className={"col-lg-9 col-md-6"}>
+                            <Col className={"col-lg-5 col-md-6"}>
                                 <h1 data-cy={`MovieTitle`}>{movieDetails.title}</h1>
                                 <p>{movieDetails.release_date}</p>
                                 <p>{movieDetails.vote_average}/10</p>
                                 <p>{movieGenres}</p>
                                 <p>{movieDetails.overview}</p>
                             </Col>
+                            <Col className={"col-lg-2 col-md-6"}>
+                                <Link href={"/movies/" + movieId + "/lists"} data-cy={"addToListButton"}>Add To List</Link>
```

Comment:
Why was this not styled as a proper button?
Why is this a link instead of a button triggering a javascript function?

Reply:
It has to open another page. I thought Link was the correct one to use in that case.